fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzerUtils.java

```diff
@@ -201,7 +201,7 @@ private void getDB(TableName tableName) {
             if (Strings.isNullOrEmpty(dbName)) {
                 dbName = session.getDatabase();
             } else {
-                if (CatalogMgr.isInternalCatalog(session.getCurrentCatalog())) {
+                if (CatalogMgr.isInternalCatalog(tableName.getCatalog())) {
```

Comment:
MetaUtils.normalizationTableName?

Reply:
it has executed normalTableName here.  it cover the case that query connector table when current_catalog is default_catalog.